x-pack/plugins/enterprise_search/server/routes/enterprise_search/index.ts

```diff
@@ -9,6 +9,7 @@ import { RouteDependencies } from '../../plugin';
 
 import { registerDocumentRoute } from './documents';
 import { registerEnginesRoutes } from './engines';
+import { registerEnginesSearchRoutes } from './engines_preview';
```

Comment:
nitpick: worth just putting it in EnginesRoutes for now as there will only be one route.